api/src/routes/challenge.ts

```diff
@@ -347,14 +348,18 @@ export const challengeRoutes: FastifyPluginCallbackTypebox = (
           completedDate: Date.now()
         };
 
-        if (challengeType === challengeTypes.multifileCertProject) {
+        if (
+          challengeType === challengeTypes.multifileCertProject ||
+          challengeType === challengeTypes.multifilePythonCertProject
+        ) {
           completedChallenge.isManuallyApproved = true;
```

Comment:
I realise this is out of the scope of this PR, but this is a new bug:

https://github.com/freeCodeCamp/freeCodeCamp/blob/fd4e275cfbd1e115a4eecb6c7d183a4f9f6d3137/api-server/src/server/boot/challenge.js#L409

```suggestion
          completedChallenge.isManuallyApproved = false;
```

Reply:
Nice catch. It could maybe use a separate discussion. If I recall, the purpose of it was to make sure the user code didn't have anything "bad" - but it could be repurposed, or dual purposed, as a way to "verify" the projects like we discussed in Dallas. In that case, we may want to add it. I did make a [somewhat related issue](https://github.com/freeCodeCamp/freeCodeCamp/issues/53929) about the display of the python projects.

Since the current API is not adding this, I think we should do the same on the new API for now. I will remove it.

Reply:
I am trying to remember the distinction between `needsModeration` and `isManuallyApproved`.

As best as I can remember:
- `needsModeration` is set to `true` if the project should be put in the moderation queue
- `isManuallyApproved` is set to `false` if the project has been moderated, and was rejected

Does that ring a bell with you?

Reply:
I found what looks to be the PR that added `needsModeration`: https://github.com/freeCodeCamp/freeCodeCamp/pull/45921 - and here's the one with `isManuallyApproved`: https://github.com/freeCodeCamp/freeCodeCamp/pull/44505

- it looks like `needsModeration` is added to the **user** object (not `completedChallenges`) if they have one or more projects that need reviewing - to make querying the users with projects that need reviewing easier.
- and it looks like `isManuallyApproved` defaults to `false` and then we set it to `true` after reviewing and approving? or after any type of review? Probably only after approving since it's in the variable name. I feel like I recall a conversation about maybe setting `isCheater` to `true` or something like that if we found a bad project.

Reply:
Ah. That does ring a bell now.